src/avfoundation.cs

```diff
@@ -12250,6 +12394,127 @@ interface AVPlayerItemTrack {
 #endif
 	}
 
+	[iOS (14,5), Mac (11,3), TV (14,5)]
+	[Flags]
+	[Native]
+	enum AVPlayerInterstitialEventRestrictions : ulong
+	{
+		None = 0,
+		ConstrainsSeekingForwardInPrimaryContent = (1 << 0),
+		RequiresPlaybackAtPreferredRateForAdvancement = (1 << 2),
+		DefaultPolicy = None
+	}
+
+	[TV (14,5), Mac (11,3), iOS (14,5)]
+	[BaseType (typeof(NSObject))]
+	[DisableDefaultCtor]
+	interface AVPlayerInterstitialEvent
+	{
+		[Static]
+		[Export ("interstitialEventWithPrimaryItem:time:templateItems:restrictions:resumptionOffset:")]
+		AVPlayerInterstitialEvent InterstitialEventWithPrimaryItem (AVPlayerItem primaryItem, CMTime time, AVPlayerItem[] templateItems, AVPlayerInterstitialEventRestrictions restrictions, CMTime resumptionOffset);
```

Comment:
Our naming guideline says that methods should have a verb. We also try to avoid any "With*" suffix, because it's usually redundant.

This means the API looks more like managed code:

```suggestion
		AVPlayerInterstitialEvent GetInterstitialEvent (AVPlayerItem primaryItem, CMTime time, AVPlayerItem[] templateItems, AVPlayerInterstitialEventRestrictions restrictions, CMTime resumptionOffset);
```

Reply:
gotcha, thanks!